src/client/entity-editor/name-section/name-section.js

```diff
@@ -256,6 +223,60 @@ class NameSection extends React.Component {
 							onChange={onDisambiguationChange}
 						/>
 					</Col>
+					<Col md={{span: 6}}>
+						<Row>
```

Comment:
Took me a little while to find the issue you mentioned with the weird margins on smaller screens:

The way these are used is that Row elements contain Col elements instead of the other way around.

The .row class has negative left and right margins which explains what we are seeing when collapsed;

I think the simple solution here is to replace these `<Row>` elements here with `<div>`s